src/main/java/frc/robot/RobotContainer.java

```diff
@@ -64,13 +62,20 @@ public RobotContainer() {
       System.out.println("config file not found");
     }
 
+    // tankSubsystem = new
+    // TankSubsystem(Integer.parseInt(config.getProperty("fLeft")),
+    // Integer.parseInt(config.getProperty("bLeft")),
+    // Integer.parseInt(config.getProperty("fRight")),
+    // Integer.parseInt(config.getProperty("bRight")));
+
+    // elevatorSubsystem = new
+    // ElevatorSubsystem(Integer.parseInt(config.getProperty("elevator_master")),
+    // Integer.parseInt(config.getProperty("elevator_follower")));

```

Comment:
Hey, you're using git. You can delete code because it's still there in the history. :)